src/Plugins/SimplnxCore/src/SimplnxCore/SurfaceNets/MMCellFlag.h

```diff
@@ -10,119 +10,116 @@
 
 #include <cstdint>
 
-class MMCellFlag
+namespace MMCellFlag
 {
-public:
-  MMCellFlag();
-  virtual ~MMCellFlag();
+// public:
+//   MMCellFlag();
+//   virtual ~MMCellFlag();
 
-  void operator=(const MMCellFlag& t)
-  {
-    m_bitFlag = t.m_bitFlag;
-  }
-
-  unsigned int getBitFlag() const
-  {
-    return m_bitFlag;
-  }
+//  void operator=(const MMCellFlag& t)
+//  {
+//    m_bitFlag = t.m_bitFlag;
+//  }
+//
+//  unsigned int getBitFlag() const
+//  {
+//    return m_bitFlag;
+//  }
 
-  enum class VertexType
-  {
-    NoVertex,
-    SurfaceVertex,
-    EdgeVertex,
-    CornerVertex
-  };
-  enum class FaceCrossingType
-  {
-    NoFaceCrossing,
-    SurfaceFaceCrossing,
-    JunctionFaceCrossing
-  };
-  enum class Face
-  {
-    LeftFace,
-    RightFace,
-    BackFace,
-    FrontFace,
-    BottomFace,
-    TopFace
-  };
-  enum class Edge
-  {
-    LeftBottomEdge,
-    RightBottomEdge,
-    BackBottomEdge,
-    FrontBottomEdge,
-    LeftTopEdge,
-    RightTopEdge,
-    BackTopEdge,
-    FrontTopEdge,
-    LeftBackEdge,
-    RightBackEdge,
-    LeftFrontEdge,
-    RightFrontEdge
-  };
+enum class VertexType
+{
+  NoVertex,
+  SurfaceVertex,
+  EdgeVertex,
+  CornerVertex
+};
+enum class FaceCrossingType
+{
+  NoFaceCrossing,
+  SurfaceFaceCrossing,
+  JunctionFaceCrossing
+};
+enum class Face
+{
+  LeftFace,
+  RightFace,
+  BackFace,
+  FrontFace,
+  BottomFace,
+  TopFace
+};
+enum class Edge
+{
+  LeftBottomEdge,
+  RightBottomEdge,
+  BackBottomEdge,
+  FrontBottomEdge,
+  LeftTopEdge,
+  RightTopEdge,
+  BackTopEdge,
+  FrontTopEdge,
+  LeftBackEdge,
+  RightBackEdge,
+  LeftFrontEdge,
+  RightFrontEdge
+};
 
-  // Set components of the cell flag. The components are determined from the
-  // cell labels of its 8 vertices, which are listed left-to-right, back-to-front,
-  // bottom-to-top (i.e., left-back-bottom vertex first and right-front-top vertex
-  // last). Clearing the cell flag encodes types NoVertex, NoFaceCrossing, and no
-  // edge or face crossings.
-  void set(int32_t cellLabels[8]);
+// Set components of the cell flag. The components are determined from the
+// cell labels of its 8 vertices, which are listed left-to-right, back-to-front,
+// bottom-to-top (i.e., left-back-bottom vertex first and right-front-top vertex
+// last). Clearing the cell flag encodes types NoVertex, NoFaceCrossing, and no
+// edge or face crossings.
+void set(unsigned int& m_bitFlag, unsigned char& m_numJunctions, int32_t cellLabels[8]);
```

Comment:
```suggestion
void set(unsigned int& bitFlag, unsigned char& numJunctions, int32_t cellLabels[8]);
```
remove member variable naming scheme in free function args